cms/models.py

```diff
@@ -165,6 +173,34 @@ class ProgramCourse(Orderable):
     ]
 
 
+class ProgramFaculty(Orderable):
+    """
+    Faculty for the program
+    """
+    program_page = ParentalKey(ProgramPage, related_name='faculty_members')
+    name = models.CharField(max_length=255, help_text='Full name of the faculty member')
+    title = models.CharField(max_length=20, default='')
```

Comment:
If these are allowed to be blank you should add a `blank=True` here, or otherwise remove `default=''` here and in the next line